spyder/api/widgets/__init__.py

```diff
@@ -405,6 +431,13 @@ def _setup(self, options=DEFAULT_OPTIONS):
                 # For widgets that use tabs, we add the corner widget using
                 # the setCornerWidget method.
                 child.setCornerWidget(self._corner_widget)
+                
+                # This is needed to ensure the corner ToolButton (hamburguer
+                # menu) is aligned with plugins that use ToolBars vs
+                # CornerWidgets
+                # See: spyder-ide/spyder#13600
+                # left, top, right, bottom
+                self._corner_widget.setContentsMargins(0, 2, 2, 0)
```

Comment:
This is needed to ensure things look ok on Mac. This may need to be a different value on a different OS.

Reply:
I was checking this and seems like there is a TODO in the history pane that could be related with the different visual between OS?: 

https://github.com/spyder-ide/spyder/blob/289fc426337ae8d8348119514a9c96af92714c84/spyder/plugins/history/widgets.py#L94-L103

Also, on WIndows, seems like the values should be `0, 0, 0, 0`. With does values the icon size is correct but there is a little misalignment (maybe since the tab is not higher because no `x` button is present in the history tab?):

![toolbars](https://user-images.githubusercontent.com/16781833/91746293-fdd31880-eb81-11ea-9423-7765df22dc97.gif)

Reply:
>TODO in the history pane that could be related to the different visual between OS?

No, that has to do with plugins using tabs. They should not worry about doing that manually and instead, we should handle that in the `tabs.py`  file directly. So it is a different issue.

Updated to 0,0,0,0 on windows